src/Common/src/Interop/Windows/HttpApi/Interop.HttpApi.cs

```diff
@@ -0,0 +1,1021 @@
+                HttpHeaderExpires = 18,   // entity-header  [section 7.1]
+                HttpHeaderLastModified = 19,   // entity-header  [section 7.1]
+
+
+                // Response Headers
+
+                HttpHeaderAcceptRanges = 20,   // response-header [section 6.2]
+                HttpHeaderAge = 21,   // response-header [section 6.2]
+                HttpHeaderEtag = 22,   // response-header [section 6.2]
+                HttpHeaderLocation = 23,   // response-header [section 6.2]
+                HttpHeaderProxyAuthenticate = 24,   // response-header [section 6.2]
+                HttpHeaderRetryAfter = 25,   // response-header [section 6.2]
+                HttpHeaderServer = 26,   // response-header [section 6.2]
+                HttpHeaderSetCookie = 27,   // response-header [not in rfc]
+                HttpHeaderVary = 28,   // response-header [section 6.2]
+                HttpHeaderWwwAuthenticate = 29,   // response-header [section 6.2]
+
+                HttpHeaderResponseMaximum = 30,
+
+
+                HttpHeaderMaximum = 41
+            }
+
+            private static string[] s_strings = {
+                    "Cache-Control",
+                    "Connection",
+                    "Date",
+                    "Keep-Alive",
+                    "Pragma",
+                    "Trailer",
+                    "Transfer-Encoding",
+                    "Upgrade",
+                    "Via",
+                    "Warning",
+
+                    "Allow",
+                    "Content-Length",
+                    "Content-Type",
+                    "Content-Encoding",
+                    "Content-Language",
+                    "Content-Location",
+                    "Content-MD5",
+                    "Content-Range",
+                    "Expires",
+                    "Last-Modified",
+
+                    "Accept-Ranges",
+                    "Age",
+                    "ETag",
+                    "Location",
+                    "Proxy-Authenticate",
+                    "Retry-After",
+                    "Server",
+                    "Set-Cookie",
+                    "Vary",
+                    "WWW-Authenticate",
+                };
+        }
+
+        private static string GetKnownHeader(HTTP_REQUEST* request, long fixup, int headerIndex)
+        {
+            if (NetEventSource.IsEnabled)
+            {
+                NetEventSource.Enter(null);
+            }
+
+            string header = null;
+
+            HTTP_KNOWN_HEADER* pKnownHeader = (&request->Headers.KnownHeaders) + headerIndex;
+
+            if (NetEventSource.IsEnabled)
+            {
+                NetEventSource.Info(null, $"HttpApi::GetKnownHeader() pKnownHeader:0x{(IntPtr)pKnownHeader:x}");
+                NetEventSource.Info(null, $"HttpApi::GetKnownHeader() pRawValue:0x{(IntPtr)pKnownHeader->pRawValue:x} RawValueLength:{pKnownHeader->RawValueLength}");
```

Comment:
Nit: just fyi, the logging implementation already formats IntPtr with 'x', so specifying it explicitly isn't needed.